demo/androidDev/app/build.gradle

```diff
@@ -65,6 +65,7 @@ jsonSchema2Pojo {
     //source = files("${sourceSets.main.output.resourcesDir}/json")
     source.setFrom files("${project.rootDir}/schema")
 
+    targetDirectoryPrefix = 'generated/source/jsonschema2dataclass'
```

Comment:
Now I understand what you mean. It's the path where to put the generated classes !
Is it the same/similar like `targetDirectory` ?
If yes, why call it xxx**Prefix** ?

Reply:
Each execution adds its index to the suffix and then each variant (for Android builds) adds its own suffix as well

Reply:
Old targetDirectory parameter worked the same way. This name clarified the usage